### DISCUSSION TITLE
Fast code navigation.

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#887] [General]
Does Lem have anything to the emacs C-[up arrow] or C-[down arrow] to be able to jump between chunks of code delimited by empty lines?

This is NOT a feature request, just curious if it already exists as I cannot fined anything in the M-x menu that behaves that  way,  Maybe Lem has a better and more effective version of that feature already?

Thanks.

**Comment by cxxxr** [comment]
The commands forward-paragraph and backward-paragraph, but assigned to `M-{` and `M-}`.

**Comment by cxxxr** [comment]
Hmmm, indeed, I would like to have a navigation function.

A similar function is the command `M-x documentation-describe-bindings`.
Also the same file is also automatically generated and placed at https://github.com/lem-project/lem/blob/main/docs/default-keybindings.md.

**Comment by aykaramba** [comment]
Aha!  There is the command!  Perfect.  Also, I should have checked the docs, apologies for wasting your time on this. 

Thanks!
